src/app/downloads/downloads.component.ts

```diff
@@ -16,15 +16,14 @@ import { isPodcastChanged, isBeginDateChanged, isEndDateChanged, isIntervalChang
   template: `
     <prx-spinner *ngIf="loading$ | async" overlay="true"
                  loadingMessage="Please wait..."></prx-spinner>
-    <router-outlet name="sidenav"></router-outlet>
-    <section class="content" *ngIf="loaded$ | async">
+    <section *ngIf="loaded$ | async">
       <metrics-menu-bar></metrics-menu-bar>
       <metrics-downloads-chart></metrics-downloads-chart>
       <metrics-downloads-table [totalPages]="totalPages"></metrics-downloads-table>
```

Comment:
I'm noting that, in mobile, the table + pager get a little funky.  (Horizontal scrolling, page-number-overflow, etc).  Will that be addressed in a different ticket/PR?

Reply:
There is a separate set of tickets to update the table.

Reply:
tables design is here https://github.com/PRX/metrics.prx.org/issues/279